src/test-helpers.tsx

```diff
@@ -480,21 +480,25 @@ export const createFakeLocation = ({
   };
 };
 
+// The funny business with jestExists below is so we can use this helper with
+// Storybook, where jest does not exist.
 export const createFakeHistory = ({
   location = createFakeLocation(),
 } = {}): History => {
+  const jestExists = typeof jest !== 'undefined';
+  const fakeCallback = jestExists ? null : () => () => null;
```

Comment:
I'd rather create a fake `jest` global object in Storybook. This does not look pretty :)

Reply:
I looked into trying to get a real instance of `jest` in Storybook and didn't have much luck, but you're right, we could just have a fake `jest`. Where would you put that?

Reply:
We could put that in the storybook config:

```diff
diff --git a/stories/setup/config.tsx b/stories/setup/config.tsx
index b25ea72..6877cf2 100644
--- a/stories/setup/config.tsx
+++ b/stories/setup/config.tsx
@@ -19,6 +19,14 @@ import '../../src/styles.scss';
 // Apply some custom styles to storybook.
 import './styles.scss';
 
+// Fake jest so that we can use test helpers in stories.
+window.jest = {
+  // @ts-ignore
+  fn() {
+    return null;
+  },
+};
+
 configureApplication();
 
 // Automatically import all files ending in *.stories.tsx
```